### PR TITLE
Main

### PR DESCRIPTION
**Pull request by Joel-Cripps**
Change to primary secondary. Branch to main.
Added todo for values that can not be changed in terraform.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
 A collection of **Node.js** modules and cloud-specific templates which support autoscale functionality for groups of FortiGate-VM instances on various cloud platforms.
 
 This project contains the code and templates for the **AliCloud Autoscale** deployment.
 For autoscale on other cloud providers see the [fortigate-autoscale](https://github.com/fortinet/fortigate-autoscale/) repository.
 
 ## Supported Platforms
-This project supports autoscale for the cloud platforms listed below. The version tag in parentheses refers to the autoscale module version included in this project.
 
-  * AliCloud (1.0.0)
+This project supports autoscale for the cloud platforms listed below. The version tag in parentheses refers to the autoscale module version included in this project.
```
Comment (klc178) [inline README.md:10]:
```suggestion
This project supports autoscale for the cloud platform listed below.
```

File: README.md [hunk 2]
```diff
@@ -1,38 +1,41 @@
...
 
+- AliCloud (1.0.0)
```
Comment (klc178) [inline README.md:12]:
```suggestion
- AliCloud
```

File: README.md [hunk 3]
```diff
@@ -1,38 +1,41 @@
 # FortiGate Autoscale
+
 A collection of **Node.js** modules and cloud-specific templates which support autoscale functionality for groups of FortiGate-VM instances on various cloud platforms.
 
 This project contains the code and templates for the **AliCloud Autoscale** deployment.
```
Comment (klc178) [inline README.md:5]:
```suggestion
This project contains the code and templates for the **FortiGate Autoscale for AliCloud** deployment.
For autoscale on **GCP** see the [fortigate-autoscale-gcp](https://github.com/fortinet/fortigate-autoscale-gcp/) repository.
For autoscale on **Amazon AWS** and **Microsoft Azure** see the [fortigate-autoscale](https://github.com/fortinet/fortigate-autoscale/) repository.

File: README.md [hunk 4]
```diff
@@ -1,38 +1,41 @@
...
 For autoscale on other cloud providers see the [fortigate-autoscale](https://github.com/fortinet/fortigate-autoscale/) repository.
```
Comment (klc178) [inline README.md:6]:
```suggestion
```

File: README.md (hunk 3 above)
Comment (jamie-pate) [reply README.md:5]: templates for ~the~ **FortiGate Autoscale for AliCloud** deployment. ?

**Comment by Joel-Cripps** [comment]
Added in Kimberly's changes, let me know!

**Pull request merged**